test/rdf-components-mocha.js

```diff
@@ -104,4 +122,30 @@ describe('rdf components', function() {
             });
         }).should.eventually.eql(cynthia);
     });
+    it("should round trip a graph through ntriples", function() {
```

Comment:
Not sure I understand this test description.  Round tripping a graph by sending data, I get.  But what is round tripping through ntriples?  Aren't the ntriples the data?